RxCocoa/Traits/ControlEvent.swift

```diff
@@ -30,9 +30,9 @@ public protocol ControlEventType : ObservableType {
 
     **It is the implementor’s responsibility to make sure that all other properties enumerated above are satisfied.**
 
-    **If they aren’t, using this trait will communicate wrong properties, and could potentially break someone’s code.**
+    **If they aren't, using this trait will communicate wrong properties, and could potentially break someone’s code.**
 
-    **If the `events` observable sequence passed into the initializer doesn’t satisfy all enumerated
+    **If the `events` observable sequence passed into the initializer doesn't satisfy all enumerated
```

Comment:
not a typo

Reply:
Not a typo really, but maybe change this for consistency?
```
ag "doesn’t" | wc -l // 2
ag "doesn't" | wc -l // 58
```

Reply:
Reverted for now, it can be made in another PR if needed